### ISSUE TITLE
Old games without sound. Samples directory?

### ISSUE THREAD

**Issue by godzil21** [libretro/mame2003-libretro#26]
Does anyone know where to put the samples directory? I'm trying to run space invaders , or turbo but they lack sound...Also Donkey Kong lacks some sounds. I've tried to put the directory samples in the same directory of this libretro dll but without luck. No sound. 

Thanks a lot!

**Comment by dankcushions** [comment]
hi! the sample zip files need to go in the /system dir/mame2003/sample/

your system dir is set in the 'directories' page of the retroarch menu

**Comment by godzil21** [comment]
Ops! Thanks a lot but I continue without sounds... I believe my problem is that I'm using an old mame2003 compiled edition for windows. I have tried to download the compiled version from retroarch but I can't find it. Also I tried to compile this version in windows but the compiler keeps crashing with the too many arguments issue. So ... I can download an updated compiled version of this emulator?

Thanks a lot again!
